tests/integration/utils/snapshots.mjs

```diff
@@ -37,4 +37,5 @@ const normalize = (inputString, { duration, filePath } = {}) => {
     .trim()
 }
 
-module.exports = { normalize }
+const exports = { normalize }
+export default exports
```

Comment:
copilot let you down?

Reply:
Lol yeah

Reply:
Actually, I'm realizing this was here because of this lint error that occurs when I just did `export default {normalize}`

<img width="1102" alt="Screenshot 2023-10-30 at 8 11 41 AM" src="https://github.com/netlify/cli/assets/5655473/3e1c9742-4934-43bc-b738-cd33ed5a8d60">

Reply:
yeah but it's already exported, so we should be fine without those lines

Reply:
🤦🏻‍♀️ 

Excuse me while I go get some caffeine ☕ 

Good shout, removing these lines now.